test/lib.bash

```diff
@@ -9,7 +9,7 @@ source "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")/hack/lib/__sou
 readonly TEARDOWN="${TEARDOWN:-on_exit}"
 export TEST_NAMESPACE="${TEST_NAMESPACE:-serverless-tests}"
 declare -a TEST_NAMESPACES
-TEST_NAMESPACES=("${TEST_NAMESPACE}" "serverless-tests2" "serverless-tests-mesh")
+TEST_NAMESPACES=("${TEST_NAMESPACE}" "serverless-tests-mesh")
```

Comment:
@mgencur is this still ok? We do not have a namespace that is `serverless-tests-mesh`?